Cant import Typeahead using 8.0.0-RC.5

Have working app.
- Update package.json to include "ngx-bootstrap": "8.0.0-RC.5" from "7.1.2".
- run npm i
- run npm start

see lots of errors all related to importing typeahead
`./src/app/my.module.ts:7:0-58 - Error: Module not found: Error: Can't resolve 'ngx-bootstrap/typeahead' in 'C:\my-file'`
Rest of ngx-bootstrap imports seem ok. Only typeahead is failing

```
@angular-devkit/architect       0.1301.2
@angular-devkit/build-angular   13.1.2
@angular-devkit/core            13.1.2
@angular-devkit/schematics      13.1.2
@angular/cli                    13.1.2
@schematics/angular             13.1.2
rxjs                            7.5.1
typescript                      4.4.4
```

--- a/src/root/package.json
+++ b/src/root/package.json
@@ -193,12 +193,12 @@
       "default": "./tooltip/fesm2020/ngx-bootstrap-tooltip.mjs"
     },
     "./typeahead": {
-      "types": "./ngx-bootstrap-typeahead.d.ts",
-      "esm2020": "./esm2020/ngx-bootstrap-typeahead.mjs",
-      "es2020": "./fesm2020/ngx-bootstrap-typeahead.mjs",
-      "es2015": "./fesm2015/ngx-bootstrap-typeahead.mjs",
-      "node": "./fesm2015/ngx-bootstrap-typeahead.mjs",
-      "default": "./fesm2020/ngx-bootstrap-typeahead.mjs"
+      "types": "./typeahead/ngx-bootstrap-typeahead.d.ts",
+      "esm2020": "./typeahead/esm2020/ngx-bootstrap-typeahead.mjs",
+      "es2020": "./typeahead/fesm2020/ngx-bootstrap-typeahead.mjs",
+      "es2015": "./typeahead/fesm2015/ngx-bootstrap-typeahead.mjs",
+      "node": "./typeahead/fesm2015/ngx-bootstrap-typeahead.mjs",
+      "default": "./typeahead/fesm2020/ngx-bootstrap-typeahead.mjs"
     },
     "./utils": {
       "types": "./utils/ngx-bootstrap-utils.d.ts",

